assets/l10n/app_en.arb

```diff
@@ -160,6 +160,31 @@
       "message": {"type": "String", "example": "Invalid format"}
     }
   },
+  "errorServerConnectionTitle": "Connection error",
+  "@errorServerConnectionTitle": {
+    "description": "Error title when the app cannot stay connected to the server."
+  },
+  "errorLoadingServerData": "Error loading server data from {url}. Will retry:\n\n{error}",
+  "@errorLoadingServerData": {
+    "description": "Dialog message for a generic unknown error from the server.",
+    "placeholders": {
+      "url": {"type": "String", "example": "http://example.com/"},
+      "error": {"type": "String", "example": "Invalid format"}
+    }
+  },
+  "errorConnectingToServer": "Failed to reach server at {url}. Will retry:\n\n{error}",
+  "@errorConnectingToServer": {
+    "description": "Dialog message when the the server cannot be reached",
+    "placeholders": {
+      "url": {"type": "String", "example": "http://example.com/"},
+      "error": {"type": "String", "example": "Network request failed: HTTP status 500"}
+    }
+  },
+  "errorReconnectingToServer": "Reconnecting to server at {url}...",
```

Comment:
How about "Reconnecting to {url}…"? It's a bit shorter, and also I think we prefer "…" over "...".

Reply:
Yeah, we can do a tiny sweep to replace "..." with "…"  in our translation strings. I saw the character used in tests and comments, but not sure why we didn't do that in `app_en.arb`.

Reply:
Opened #911.